### ISSUE TITLE
Elytra with fireworks in rain/snow causes kick

### ISSUE THREAD

**Issue by Bobcat00** [Updated-NoCheatPlus/NoCheatPlus#106]
### Full output of /ncp version
```
#### Server ####
3019-Spigot-bff5261-ee5006d ~MC: 1.16.5~
  detected: 1.16.5
#### NoCheatPlus ####
Plugin: 3.17-SNAPSHOT-Updated-b92
MCAccess: 1.13-1.16|? / Bukkit-API
Features:
  blocks: BlocksMC1_4 | BlocksMC1_5 | BlocksMC1_6_1 | BlocksMC1_7_2 | BlocksMC1_8 | BlocksMC1_9 | BlocksMC1_10 | BlocksMC1_11 | BlocksMC1_12 | BlocksMC1_13 | BlocksMC1_14 | BlocksMC1_15 | BlocksMC1_16 | MCAccessBukkitModern
  checks: FastConsume | Gutenberg | HotFixFallingBlockPortalEnter | AttackFrequency | FlyingFrequency | KeepAliveFrequency
  defaults: pvpKnockBackVelocity
  packet-listeners: UseEntityAdapter | MovingFlying | OutgoingPosition | KeepAliveAdapter | WrongTurnAdapter | NoSlow | Fight
Hooks: AllViolations~NCP~ 1.0 | ViolationFrequency~NCP~ 1.0
#### Related Plugins ####
ProtocolLib v4.6.0 | ViaVersion v3.2.1
```

### Describe the issue
A player reports "the server kicks me for firework elytraing in the rain or snow." He says it works OK in the sun.

### How to reproduce the issue
See above.

### Extra links/Videos (Including debug logs)
Although it says CreativeFly below, the player is in Survival mode.
```
2021-03-14 01:20:52 [INFO] [NoCheatPlus] Mr_Microwave_Man failed CreativeFly: tried to perform an illegal move from (1753.01, 69.50, 584.13) to (1753.83, 69.66, 584.01) over a distance of (0.84) blocks, subchecks triggered> (bunnyhop+e_vasc+e_vdiff+e_hspeed+hdist+vdist) (VL945)
2021-03-14 01:20:52 [INFO] [NCP] Mr_Microwave_Man failed CreativeFly: performed a move that deviates from the model (Model: jetpack.elytra, Tags: bunnyhop+e_vasc+e_vdiff+e_hspeed+hdist+vdist) (VL1117)
2021-03-14 01:20:53 [INFO] [NCP] Mr_Microwave_Man failed CreativeFly: performed a move that deviates from the model (Model: jetpack.elytra, Tags: bunnyhop+e_hspeed+hdist) (VL2037)
2021-03-14 01:20:53 [INFO] NCP: Issued history and data removal (MOVING_CREATIVEFLY): Mr_Microwave_Man
2021-03-14 01:20:53 [INFO] [NoCheatPlus] Mr_Microwave_Man failed CreativeFly: performed a move that deviates from the model (Model: jetpack.elytra, Tags: bunnyhop+e_hspeed+hdist) (VL2037)
2021-03-14 01:20:53 [INFO] Mr_Microwave_Man lost connection: (!) Moved unexpectedly.
2021-03-14 01:20:53 [INFO] Mr_Microwave_Man left the game
2021-03-14 01:20:53 [INFO] [NoCheatPlus] (CONSOLE) Kicked Mr_Microwave_Man : §c§l(!)§7 Moved unexpectedly.
```

### Any possible config options changed or plugins that may cause interference?
None that I'm aware of.

**Comment by Lysandr0** [comment]
Closing in favour of #132.
